CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
+## 1.1.0 
+### Features
+1. [#95](https://github.com/influxdata/influxdb-client-go/issues/95)  Authorization API
```

Comment:
we should be better referencing pull request as influxdata/influxdb does

Reply:
Sure. Fixed.